test/messages.ml

```diff
@@ -0,0 +1,22 @@
+(** Utils function shared by the different tests modules *)
```

Comment:
I think we could more accurately describe this module

Reply:
Done.